app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,25 @@
+class PetApplicationsController < ApplicationController
+  def index
+    pet = Pet.find(params["pet_id"])
+    @applicants = pet.applications
+  end
+
+  def update
+    pet = Pet.find(params[:pet_id])
```

Comment:
Is there a way to reduce the number of lines in this method? Maybe make a helper method?